src/index.ts

```diff
@@ -18,17 +18,17 @@ import { mathInputRules } from "./plugins/math-inputrules";
 import { editorSchema } from "./math-schema";
 import { MathView, ICursorPosObserver } from "./math-nodeview";
 import { mathBackspace } from "./plugins/math-backspace";
-import { mathPlugin } from "./math-plugin";
+import { mathPlugin, createMathView } from "./math-plugin";
```

Comment:
My schema was different than yours, so I actually exported the createMathView -- which might be better? But the css and things depended on the naming of the nodes.